common/src/main/java/com/microsoft/identity/common/internal/dto/AccessTokenRecord.java

```diff
@@ -86,7 +93,7 @@
      * utilize the access token to make a protected resource request (along with type-specific
      * attributes).
      */
-    @SerializedName(ACCESS_TOKEN_TYPE)
+    @SerializedName(value = TOKEN_TYPE, alternate = ACCESS_TOKEN_TYPE)
```

Comment:
❤

Reply:
Isn't that slick?? I didn't know Gson could do that until.... now 😅